templates/block-js.mustache

```diff
@@ -0,0 +1,36 @@
+( function( wp ) {
+	var el = wp.element.createElement;
+	var __ = wp.i18n.__;
+
+	// Visit https://wordpress.org/gutenberg/handbook/block-api/ to learn about Block API
+	wp.blocks.registerBlockType( '{{namespace}}/{{slug}}', {
+		title: __( '{{title_ucfirst}}' ),
+
+		{{#dashicon}}
+		icon: '{{dashicon}}',
+
+		{{/dashicon}}
+		category: '{{category}}',
+
+		// Remove to make block editable in HTML mode.
+		supportHTML: false,
+
+		edit: function( props ) {
+			return el(
+				'p',
+				{ className: props.className },
+				__( 'Replace with your content!', '{{textdomain}}' )
```

Comment:
Would it be helpful to have a more substantive example?

Reply:
I think this example is good, maybe we should comment it differently. @mtias, do you have some thoughts on that?